python/nano/src/bigdl/nano/quantization/neural_compressor/metric/__init__.py

```diff
@@ -13,10 +13,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+from neural_compressor.metric import METRICS
 
 from .metrics import *
 
-METRICS = {
-    'pytorch': PytorchINCMetric,
-    'tensorflow': TensorflowINCMetric
+METRICS('pytorch').register('PytorchINCMetric', PytorchINCMetric)
+METRICS('tensorflow').register('TensorflowINCMetric', TensorflowINCMetric)
+
```

Comment:
Where `PytorchINCMetric` and `TensorflowINCMetric` are  defined?

Reply:
https://github.com/intel-analytics/BigDL/blob/branch-2.0/python/nano/src/bigdl/nano/quantization/neural_compressor/metric/metrics.py#L19-L40